### PR TITLE
Fix some minor typos and reference to "develop" branch

### PR DESCRIPTION
**Pull request by necrophonic**
- Fix spelling of "license" in readme
- Remove reference to "develop" branch in contributing

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 Contributing to Survey Data Collection repositories
 ===================================================
 
-* We use git-flow - create a feature branch from develop, e.g. feature/new-feature
+* We use github-flow - create a branch from master, e.g.:
+    * feature/new-feature
+    * fix/some-bug
 * Pull requests must contain a succinct, clear summary of what the user need is driving this feature change
 * Ensure your branch contains logical atomic commits before sending a pull request - follow the [alphagov Git styleguide](https://github.com/alphagov/styleguides/blob/master/git.md)
 * You may rebase your branch after feedback if it's to include relevant updates from the develop branch. We prefer a rebase here to a merge commit as we
```
Comment (ajmaddaford) [inline CONTRIBUTING.md]: This line still refers to develop

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
 ===================================================
 
-* We use git-flow - create a feature branch from develop, e.g. feature/new-feature
+* We use github-flow - create a branch from master with a short, concise, name, e.g. (nb. there are no specific prefixes):
+    * add-user-auth
+    * fix-ioloop-bug
+    * update-cryptography-version
 * Pull requests must contain a succinct, clear summary of what the user need is driving this feature change
 * Ensure your branch contains logical atomic commits before sending a pull request - follow the [alphagov Git styleguide](https://github.com/alphagov/styleguides/blob/master/git.md)
 * You may rebase your branch after feedback if it's to include relevant updates from the develop branch. We prefer a rebase here to a merge commit as we
```
Comment (ajmaddaford) [inline CONTRIBUTING.md]: This still references develop rather than master

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,7 +1,10 @@
 Contributing to Survey Data Collection repositories
 ===================================================
 
-* We use git-flow - create a feature branch from develop, e.g. feature/new-feature
+* We use github-flow - create a branch from master with a short, concise, name, e.g. (nb. there are no specific prefixes):
```
Comment (ajmaddaford) [inline CONTRIBUTING.md]: I think we should link to http://scottchacon.com/2011/08/31/github-flow.html as it does a good job of explaining some of the reasoning behind github-flow

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

**Comment by necrophonic** [comment]
Requested changes done

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**